.circleci/config.yml

```diff
@@ -40,12 +30,12 @@ aliases:
 executors:
   node-executor:
     docker:
-      - image: circleci/node:13
+      - image: circleci/node:13.8
```

Comment:
nit: it should be okay to unpin node.js version.

Reply:
Circle CI doesn't support 13.11 yet: https://circleci.com/docs/2.0/circleci-images/#nodejs

Reply:
TODO: I will wait to merge this PR until it's supported.

Reply:
Looks like CircleCI support 13.11 and 13.12 now.